src/Hyperwallet/Response/ListResponse.php

```diff
@@ -61,6 +84,33 @@ public function getData() {
         return $this->data;
     }
 
+    /**
+     * Get the Total number of Model's
+     *
+     * @var int
+     */
+    public function getLimit() {
+        return $this->limit;
+    }
+
+    /**
+     * Get Has Next Page of Model's
+     *
+     * @var boolean
+     */
+    public function getHasNextPage() {
+        return $this->hasNextPage;
+    }
+
+    /**
+     * Get Has Previous Page of Model's
+     *
+     * @var boolean
+     */
+    public function getHasPreviousPage() {
+        return $this->hasPreviousPage;
+    }
+
```

Comment:
offset and count fields/methods can be removed

Reply:
This Count and Offset method using array iteration propose , so we need those methods.